blockchain/standalone/pow_test.go

```diff
@@ -273,9 +274,9 @@ func TestCheckProofOfWorkRange(t *testing.T) {
 			t.Errorf("%q: unexpected err -- got %v, want nil", test.name, err)
 			continue
 		} else if test.err != nil {
-			if !IsErrorCode(err, test.err.(RuleError).ErrorCode) {
+			if !errors.Is(err, test.err) {
```

Comment:
All of this can just be changed to:

```Go
		err := CheckProofOfWorkRange(test.bits, powLimit)
		if !errors.Is(err, test.err) {
			t.Errorf("%q: unexpected err -- got %v, want %v", test.name, err,
				test.err)
			continue
		}
```